### ISSUE TITLE
in STS approximate inference for models with non Gaussian observations, should the observation noise scale be included?

### ISSUE THREAD

**Issue by jeffpollock9** [tensorflow/probability#1148]
In [this example](https://www.tensorflow.org/probability/examples/STS_approximate_inference_for_models_with_non_Gaussian_observations) a linear Gaussian state space model is used over unconstrained Poisson rates. I'm wondering if this model is actually over-parameterised since it includes observation noise when the unconstrained rates aren't actually observed? That means that it's hard (impossible?) to differentiate between noise in the level/underlying processes and the "observation noise" in this model. To fix, I think it is as easy as changing:

``` python
def build_model(approximate_unconstrained_rates):
  trend = tfp.sts.LocalLinearTrend(
      observed_time_series=approximate_unconstrained_rates)
  return tfp.sts.Sum([trend],
                     observed_time_series=approximate_unconstrained_rates)
```

to 

``` python
def build_model(approximate_unconstrained_rates):
  trend = tfp.sts.LocalLinearTrend(
      observed_time_series=approximate_unconstrained_rates)
  return trend
```

which seems to run fine and look sensible, also the inference with HMC now runs in about 20 seconds instead of around 50.

I'm not 100% sure about this so any input would be great.

Thanks!

**Comment by junpenglao** [comment]
I think this make sense as well - NOT adding an additional error term also more inline with the GLM formulation, where the observation error is "explained" by the poisson likelihood.
@davmre

**Comment by davmre** [comment]
Yeah, I'm not sure what I was thinking when I wrote that example, but I agree there's no obvious reason to include the observation noise scale parameter. For what it's worth, I believe the parameter is identifiable: the extra degree of freedom should allow for separate control of the likelihood mean and variance, where a Poisson likelihood on its own has mean equal to its variance. But there are probably more sensible ways to get that control if you want it, e.g., using a negative binomial likelihood.

Since it speeds up inference, this seems worth fixing. I'll push an updated version in the next couple of days.

**Comment by jeffpollock9** [comment]
many thanks for the replies, @junpenglao and @davmre. I think what you both said makes sense. I also ran the notebook a few more times in google colab to check run times and they were quite varied so please don't put much trust into my claim of "runs in about 20 seconds instead of around 50."

I'll leave this open but feel free to close if you decide to not make any changes or for whatever other reason.
